### DISCUSSION TITLE
Can I replace the Vanna icon next to the chat questions?

### DISCUSSION THREAD

**Discussion by vaultagegroup** [vanna-ai/vanna#424] [Q&A]
Can I replace the Vanna icon next to the chat questions?   
![image](https://github.com/vanna-ai/vanna/assets/5218780/638595aa-326a-4f77-b64a-247372243003)

**Comment by Mioooooo** [comment]
Yes, it can be replaced,
![image](https://github.com/vanna-ai/vanna/assets/37464936/30d3f22c-92da-48fe-836e-cb453d275a1b)

**Reply by rukasudev** [reply]
Hi @Mioooooo! I think this attribute is only to the left side logo...
Is really possible to change the icon next to the chat? I searched everywhere but couldn't find it :(

**Reply by plc1220** [reply]
having the same request too

**Reply by lolbus** [reply]
This is for the top left logo, not chat icon.

**Comment by Mioooooo** [comment]
There are many attributes that can be customized, and vanna’s documentation is very detailed.

**Reply by lolbus** [reply]
I spent 2 days using sonnet 3.5 to help me modify this. There are no direct attributes to change the icon.

At first sonnet3.5 recommend me to gain access to frontend source code, which is not possible since VannaAI frontend is closed source. 

Then we worked together endlessly on the mimified code for 2 days, for some reason despite all the hard intercept to prevent the interface from connecting to vanna.ai domain, it still ends up loading vanna.svg as the chat icon. 

Frustrating max.

**Reply by OsqY** [reply]
If it helps, you can access to the frontend to the flask app: https://github.com/vanna-ai/vanna-flask, here you can tune the frontend, but you can only change it through a minified JavaScript file. The other way is you can only use the API that is being used in the app.py file and from there you can use your own frontend and access those API routes through your frontend.

**Comment by k-e-r-n-e-l-p-a-n-i-c** [comment]
@vaultagegroup , @lolbus  -  I believe you would have to find the appropriate function in this [file](https://github.com/vanna-ai/vanna-flask/blob/main/static/assets/index-d29524f4.js)  and replace the below code 

```
function CA(E) {
    let e, i;
    return {
        c() {
            e = f("span");
            const img = f("img");
            // Set image attributes
            a(img, "src", "/static/<your-image>"); // Replace with your image path
            a(img, "alt", "Logo");
            a(img, "width", "100%");
            a(img, "height", "100%");
            a(e, "class", i = "flex-shrink-0 w-[2.375rem] h-[2.375rem] " + E[0]);
            l(e, img);
        },
        m(_, c) {
            V(_, e, c);
        },
        p(_, [c]) {
            c & 1 && i !== (i = "flex-shrink-0 w-[2.375rem] h-[2.375rem] " + _[0]) && a(e, "class", i);
        },
        i: j,
        o: j,
        d(_) {
            _ && Y(e);
        }
    }
}
```
